modules/openapi-json-schema-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing-with-aws-sigv4.yaml

```diff
@@ -0,0 +1,265 @@
+openapi: 3.0.0
```

Comment:
If you are making your own spec for an example can you cut it down to one endpoint with the new security scheme and one without it?

Reply:
Good idea, I had just copied a chunk of the original one, but had not paired it down to what I needed.

I have now added a test for the `get` call without the security set, however I am getting a strange issue. I was wondering if you could shed some light on it. 

I copied the test from https://github.com/openapi-json-schema-tools/openapi-json-schema-generator/blob/master/samples/openapi3/client/petstore/python/tests_manual/test_paths/test_pet_pet_id/test_get.py

However for some reason the model isn't being generated with the paired down spec, however in both paths it still references the Pet schema as a response type. Instead I am getting a `DynamicSchema` object type as a response.

Reply:
What do you mean by being generated? Files being written? DynamicSchema instances are expected. Validated instances are made of dynamic classes (DynamicSchema) that subclass the validated classes.

Reply:
Yes so when I generated the sample, it didnt make the Pet Model. As you can see below trying to reference it in the test fails, however that schema is defined in the openapi yaml.

```
ImportError while importing test module '/home/jblakeney/work/dev/openapi-json-schema-generator/samples/openapi3/client/petstore/python_aws_sigv4/tests_manual/test_paths/test_pet_pet_id/test_get.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests_manual/test_paths/test_pet_pet_id/test_get.py:16: in <module>
    from petstore_api.model.pet import Pet
E   ModuleNotFoundError: No module named 'petstore_api.model'
```

Reply:
You need to rebase on the 2_0_0 branch. Models are not there. There are in components.schema

Reply:
Ahh perfect, that fixes it. I should of looked at the test in the code, and not just the link you provided as that was the PR! My bad!